tests/samplers_tests/test_partial_fixed.py

```diff
@@ -0,0 +1,73 @@
+from typing import Union
+
+import pytest
+
+import optuna
+from optuna.samplers import PartialFixedSampler
+from optuna.samplers import RandomSampler
+from optuna.trial import Trial
+
+
+def test_params_identity() -> None:
+    def objective(trial: Trial) -> float:
+        x = trial.suggest_uniform("x", -10, 10)
+        y = trial.suggest_uniform("y", -10, 10)
+        return x ** 2 + y ** 2
+
+    study_0 = optuna.create_study()
```

Comment:
I think this "_" can be removed.
```suggestion
    study0 = optuna.create_study()
```